config/routes.rb

```diff
@@ -5,6 +5,11 @@
   post '/login', to: "sessions#create"
   get '/breeds', to: "breeds#index"
   get '/breeds/:name', to: "breeds#show", as: "breed"
-  get '/cats/:id', to: "cats#show"
+  get '/cats/:id', to: "cats#show", as: "cat"
 
+  scope '/admin' do
+    get '/inventory', to: "home#show"
+    get '/breeds/:id/edit', to: "breeds#edit", as: "edit_breed"
+    get '/cats/:id/edit', to: "cats#edit", as: "edit_cat"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.